Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -175,6 +175,15 @@ static String processFileAndExtractZipArchive(const String& path)
                 return;
             }
 
+            NSArray<NSString *> *contentsOfTemporaryDirectory = [NSFileManager.defaultManager contentsOfDirectoryAtPath:temporaryDirectory error:nil];
+            if (contentsOfTemporaryDirectory.count == 1) {
+                NSURL *temporaryDirectoryURL = [NSURL URLWithString:temporaryDirectory];
+                NSString *temporaryDirectorySubdirectoryPath = [temporaryDirectoryURL URLByAppendingPathComponent:contentsOfTemporaryDirectory.firstObject].path;
+
+                if ([NSFileManager.defaultManager fileExistsAtPath:temporaryDirectorySubdirectoryPath isDirectory:&isDirectory] && isDirectory)
```

Comment:
```suggestion
                auto *temporaryDirectorySubdirectoryPath = [temporaryDirectory stringByAppendingPathComponent: contentsOfTemporaryDirectory.firstObject];
                if ([NSFileManager.defaultManager fileExistsAtPath:temporaryDirectorySubdirectoryPath isDirectory:&isDirectory] && isDirectory)
```

Reply:
You could also get away with `subdirectoryPath` for the var name, since it is nested.